src/scss/_mixins.scss

```diff
@@ -190,31 +187,37 @@
 @mixin _oCookieMessageAlternative() {
 	.o-cookie-message.o-cookie-message--alternative {
 		.o-cookie-message__button {
-			//TODO: revisit argument names in o-buttons
-			@include oButtonsTheme((
-				accent: oColorsGetUseCase(o-cookie-message-alternative-button, background),
-				background: oColorsGetUseCase(o-cookie-message-alternative-button, text),
-				hover: oColorsGetUseCase(o-cookie-message-alternative-button, hover),
-				colorizer: primary
-			));
+			@include oButtonsContent((
+				theme: (
+					accent: oColorsGetUseCase(o-cookie-message/alternative-button, background),
+					background: oColorsGetUseCase(o-cookie-message/alternative-button, text),
+					hover: oColorsGetUseCase(o-cookie-message/alternative-button-hover, background),
+					colorizer: primary
```

Comment:
the button type is given with `type` key in the root of this options map, `accent` is `color` and `background` is `context`